TACC/Core-Portal
# WP-104: phantom entry in ViewTab when user removed

## Overview
When a user was selected in the View Team tab, then the same user was removed in the Manage Team tab, when you clicked back to View Team, that user's info was still here, even if they were removed. 

## Related

* [WP-104](https://jira.tacc.utexas.edu/browse/WP-104)

## Changes
Added a conditional statement to reset the AllocationContactCard back to it's default state where it displays "Click on a user’s name to view their allocation usage." when the tab is switched back to View Team from Manage Team.


## Testing
Find a Allocation that your are PI or manager on (you'll need to see the Manage Team tab)
1. In the View Team tab, select a user
2. Click on Manage Team
3. From the Manage Team tab, remove the previously selected user.
4. Click on View Team
5. Notice that the previously selected user info is not there. Instead it should display "Click on a user’s name to view their allocation usage."

## UI

https://github.com/TACC/Core-Portal/assets/35277477/46296abb-f220-4bb7-a2a2-166eb8ab8d79



## Notes
